### ISSUE TITLE
REQUEST: @DeepanshuA as approver in dapr/js-sdk

### ISSUE THREAD

**Issue by shubham1172** [dapr/community#312]
### GitHub Username

@DeepanshuA as an approver in dapr/js-sdk

### List of contributions

Top highlights
1. https://github.com/dapr/js-sdk/pull/462 (Author)
1. https://github.com/dapr/js-sdk/pull/437 (Author)
1. https://github.com/dapr/js-sdk/pull/422 (Author)
1. https://github.com/dapr/js-sdk/pull/340 (Author)
1. https://github.com/dapr/js-sdk/pull/334 (Author)
1. https://github.com/dapr/js-sdk/pull/346 (Reviewer)
1. https://github.com/dapr/js-sdk/pull/381 (Reviewer)
1. https://github.com/dapr/js-sdk/pull/443 (Reviewer)

### Nomination

Nominated by @shubham1172. @DeepanshuA has authored and reviewed some crucial PRs for JS-SDK, and has been a core contributor to the JS-SDK community calls.

**Comment by XavierGeerinck** [comment]
+1 from my side, @DeepanshuA has become a valuable member of the JS SDK with active contributions and always being present!

**Comment by artursouza** [comment]
+1

**Comment by yaron2** [comment]
Using this as a reminder of our new acceptance rules for approvers: https://github.com/dapr/community/blob/master/community-membership.md#requirements-1.

TL;DR you now have two maintainer approvals which means @DeepanshuA is accepted as an approver.

**Comment by shubham1172** [comment]
@yaron2 could you please add @DeepanshuA as an approver? Their permissions are not reflecting yet.
<img width="383" alt="image" src="https://github.com/dapr/community/assets/22132944/403c96a1-3263-4e5a-91e0-b4014ec8c9d1">

**Comment by yaron2** [comment]
> @yaron2 could you please add @DeepanshuA as an approver? Their permissions are not reflecting yet. <img alt="image" width="383" src="https://user-images.githubusercontent.com/22132944/241205477-403c96a1-3263-4e5a-91e0-b4014ec8c9d1.png">

Added
